src/Optimism.sol

```diff
@@ -3,6 +3,57 @@ pragma solidity >=0.8.0;
+    /*** Token Addresses                                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant SUSDS = 0xb5B2dc7fd34C249F4be7fB1fCea07950784229e0;
+    address internal constant USDC  = 0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85;
+    address internal constant USDS  = 0x4F13a96EC5C4Cf34e442b46Bbd98a0791F20edC3;
+
+    /******************************************************************************************************************/
+    /*** Bridging Addresses                                                                                         ***/
+    /******************************************************************************************************************/
+
+    address internal constant CCTP_TOKEN_MESSENGER = 0x2B4069517957735bE00ceE0fadAE88a26365528f;
+
+    address internal constant SKY_GOV_RELAY = 0x10E6593CDda8c58a1d0f14C5164B376352a55f2F;
+    address internal constant TOKEN_BRIDGE  = 0x8F41DBF6b8498561Ce1d73AF16CD9C0d8eE20ba6;
+
+    /******************************************************************************************************************/
+    /*** PSM Addresses                                                                                              ***/
+    /******************************************************************************************************************/
+
+    address internal constant PSM3 = 0xe0F9978b907853F354d79188A3dEfbD41978af62;
+
+    /******************************************************************************************************************/
+    /*** Spark Liquidity Layer Addresses                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant ALM_CONTROLLER  = 0x1d54A093b8FDdFcc6fBB411d9Af31D96e034B3D5;
+    address internal constant ALM_PROXY       = 0x876664f0c9Ff24D1aa355Ce9f1680AE1A5bf36fB;
+    address internal constant ALM_RATE_LIMITS = 0x6B34A6B84444dC3Fc692821D5d077a1e4927342d;
+
+    address internal constant ALM_FREEZER  = 0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431;
+    address internal constant ALM_RELAYER  = 0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB;
+    address internal constant ALM_RELAYER2 = 0x8Cc0Cb0cfB6B7e548cfd395B833c05C346534795;
+
+    /******************************************************************************************************************/
+    /*** Governance Relay Addresses                                                                                 ***/
+    /******************************************************************************************************************/
+
+    address internal constant SPARK_EXECUTOR = 0x205216D89a00FeB2a73273ceecD297BAf89d576d;
```

Comment:
SPARK_RECEIVER is SUBMISSION_ROLE
Deployer is not admin
SPARK_EXECUTOR is DEFAULT_ADMIN_ROLE on itself
```
forge verify-bytecode 0x205216D89a00FeB2a73273ceecD297BAf89d576d Executor --constructor-args 0 604800
Verifying bytecode for contract Executor at address 0x205216D89a00FeB2a73273ceecD297BAf89d576d
Creation code matched with status full
Runtime code matched with status full
```

Verify params
```
delay = 0
gracePeriod = 7 days
```